lib/bamboo.js

```diff
@@ -470,6 +470,53 @@ module.exports = function(host, username, password) {
             }
         )
     }
+    
+    /**
+     * creates a plan branch in Bamboo 
+     * 
+     * this call seems to be not documented, so we relay on some forum data:
+     * https://answers.atlassian.com/questions/215164/bamboo-rest-api-to-create-branches
+     * 
+     * note that for this to work, a repository needs to be link to Bamboo plan, since its not well documented
+     * I dont know how to handle multiple repositories  
+     * 
+     * On success - callback(null, true)
+     * On failure - callback(error)
+     *
+     * @param   {String} planKey - "PROJECT_KEY-PLAN_KEY"
+     * @param   {String} bambooBranchName - the name that will be displayed in Bamboo for the branch
+     * @param   {String} vcsBranchName - the branch name in the repository, this parameter depends on the version control used,
+     * in git its something like "refs/heads/BRANCH_NAME" - you can find it by going to .git/refs/heads folder in your file system 
+     * @param   {Object} params
+     * @param   callback
+     */
+    Bamboo.prototype.createBranchPlan = function(planKey, bambooBranchName, vcsBranchName, callback) {
+        var successResult = true;
+        var self = this,
+            planUri = self.host + "/rest/api/latest/plan/" + planKey + "/branch/" + bambooBranchName + ".json?vcsBranch=" + vcsBranchName;
+        
+        request.put(
+            planUri,
+            {},
+            function(error, response, body) {
+                
+                if (response.statusCode !== 200) {
+                   return new Error(body);
+                }
+                
+                if (error) {
+                    callback(error);
+                }
+                try {
+                    var json = JSON.parse(body.toString("utf-8", 0));
```

Comment:
`String.prototype.toString()` have no params, https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/toString

Please, just use `JSON.parse(body)`.

You do not need to cast `body` to string, because server response always just a string